### PR TITLE
Allow user-defined `JAVA_OPTS` in the startup script

### PR DESCRIPTION
**Pull request by Snile826**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

Allowing JAVA_OPTS to read environment variables instead of hard coding helps to use systemd management services.

### CODE REVIEW

**Comment by Snile826** [comment]
- The shebang must be on the first line. Delete blanks and move comments.shellcheck(SC1128)
- Use $(...) notation instead of legacy backticked `...`.shellcheck(SC2006)
- Use 'cd ... || exit' or 'cd ... || return' in case cd fails.shellcheck(SC2164)

File: CHANGES.md [hunk 1]
```diff
@@ -9,12 +9,11 @@ Release Notes.
...
 #### Java Agent
 * Remove invalid mysql configuration in agent.config.
-* Add net.bytebuddy.agent.builder.AgentBuilder.RedefinitionStrategy.Listener 
-  to show detail message when redifine errors occur
+* Add net.bytebuddy.agent.builder.AgentBuilder.RedefinitionStrategy.Listener to show detail message when redifine errors occur
 
 
 #### OAP-Backend
-
+* Repair bin/*.sh script according to shellcheck
```
Comment (kezhenxu94) [inline CHANGES.md:16]:
```suggestion
* Allow user-defined `JAVA_OPTS` in the startup script.
```

**Review by kezhenxu94 — COMMENTED** [review]
LGTM, only one nit

**Review by kezhenxu94 — APPROVED** [review]

**Comment by Snile826** [comment]
Foreground startup helps containerization.

**Pull request merged**